examples/pipeline_wavernn/utils.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import os
+import shutil
+from collections import defaultdict, deque
+
+import torch
+
+
+class MetricLogger:
+    r"""Logger for model metrics
+    """
+
+    def __init__(self, group, print_freq=1):
+        self.print_freq = print_freq
+        self._iter = 0
+        self.data = defaultdict(lambda: deque(maxlen=self.print_freq))
+        self.data["group"].append(group)
+
+    def __call__(self, key, value):
```

Comment:
let's replace this by `__setitem__` so that we can do `metric["epoch"] = epoch` instead :)

Reply:
We could add `__getitem__` as
```python
def __getitem__(self, key):
  return self.data[key][-1]
```
to get the last value. But let's not since we are not using it in the code.